csi/util.go

```diff
@@ -247,8 +250,16 @@ func ensureMountPoint(targetPath string, mounter mount.Interface) (bool, error)
 }
 
 func unmount(targetPath string, mounter mount.Interface) error {
-	logrus.Debugf("trying to unmount potential mount point %v", targetPath)
-	err := mounter.Unmount(targetPath)
+	var err error
+
+	forceUnmounter, ok := mounter.(mount.MounterForceUnmounter)
```

Comment:
When will the input mount be `MounterForceUnmounter`?

Reply:
The `UnmountWithForce` is implemented on Linux ([code](https://github.com/longhorn/longhorn-manager/blob/master/vendor/k8s.io/mount-utils/mount_linux.go#L313)), so we can assert `Mounter` as `MounterForceUnmounter` ([code](https://github.com/longhorn/longhorn-manager/blob/master/vendor/k8s.io/mount-utils/mount.go#L84)).

Reply:
Then by default will the unmounter be considered as the force unmounter?

Reply:
> Then by default will the unmounter be considered as the force unmounter?

Yeah, your idea is fine, because LH only supports linux for now.
Do you think we should use forceUnmounter by default? Or keep the general method?

Reply:
I mean, will this lead to the input unmounter be automatically converted to the force unkounter? If No, when will the input unmounter be the force one?

Reply:
It just depends on how `mounter` is created. There is no any magic conversion, but rather if `mounter` implements unmounter interface.